File: Practica 1/Analisis_exploratorio.r
#Cambiar directorio
setwd("/Users/mg454/OneDrive/Escritorio/Metodo de analisis de datos")

#Ver directorio actual
cat("Directorio actual:", getwd(),"\n")

#Ver archivos
cat("Archivos en carpeta:\n")
print(list.files())

#Cargar dataset
df <- read.csv("SP.csv")

#Dimensiones
cat("Dimensiones:\n")
print(dim(df))

#Informacion general
cat("\nInformacion del datasheet:\n")
print(str(df))

#Primeras filas 
cat("\nPrimeras 10 filas del datasheet:\n")
print(head(df,10))

#Estadisticas descriptivas
cat("\nEstadisticas descriptivas del dataset:\n")
print(summary(df))

File: Practica 1/Analisis_exploratorio.py
import os

os.chdir("/Users/mg454/OneDrive/Escritorio/Metodo de analisis de datos")
print("Directorio actual:", os.getcwd())
print("Archivos en carpeta:", os.listdir())

import kagglehub
import pandas as pd

#Cargar dataset SP
df = pd.read_csv('SP.csv')

#Exploracion inicial

#Ver dimensiones del datasheet
print("Dimensiones:", df.shape)

#Informacion general del datasheet
print("Informacion del datasheet:\n", df.info())

#Ver primeras filas
print("Primeras 10 filas del datasheet:\n", df.head(10))

#Estadisticas descriptivas
print("Estadisticas descriptivas:\n", df.describe())